src/components/Button/Button.tsx

```diff
@@ -1,24 +1,43 @@
 import React, { useState } from "react";
 
-import { testIds } from "../Button/testIds.js";
+import { testIds } from "./testIds";
 
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faCheck, faTimes } from "@fortawesome/free-solid-svg-icons";
 
 import styles from "./Button.module.css";
 
-export const Button = ({ onClick, withConfirm = false, text }) => {
+export type IProps = {
+  onClick: () => void;
+  text: string;
+  withConfirm?: boolean;
+  customClass?: string;
```

Comment:
custom class - is a very bad practice. You should calculate class on an element depending on props

Reply:
Also, you have  more props, but where are more tests?)